pkgs/development/libraries/ogre/default.nix

```diff
@@ -8,11 +8,11 @@
 , nvidia_cg_toolkit }:
 
 stdenv.mkDerivation {
-  name = "ogre-1.9.0";
+  name = "ogre-1.9-hg";
```

Comment:
It's a different version. Ogre doesn't have a great release process, and I couldn't find a newer 1.9.x than 1.9.0, even though their 1.9 branch on Bitbucket has received numerous fixes, one of which is required to build Gazebo.